Martinh/reorg create tokens guide (#220)
wormhole-foundation/wormhole-docs: tutorials/by-product/multichain-assets/index.md

---

> title: Multichain Assets Guide

Reviewer: ```suggestion
title: Multichain Assets Tutorials
```

> description: description: Learn how to create multichain tokens, register wrapped tokens, and manage their metadata for seamless multichain interoperability.

Reviewer: ```suggestion
description: Explore comprehensive, step-by-step tutorials on how to register, manage, and work with multichain assets within the Wormhole ecosystem.
```

---

> # Multichain Assets Guide

Reviewer: ```suggestion
# Multichain Assets
```

This needs to cater to potentially more than one guide

> This section provides step-by-step instructions for creating multichain tokens, registering them on multiple blockchains, and ensuring their metadata is correctly configured. This guide will ensure your token is visible, usable, and trusted across networks.

Reviewer: ```suggestion
Multichain assets, often represented as wrapped tokens, enable seamless cross-chain interoperability. This section provides step-by-step tutorials for registering, managing, and working with these assets across different blockchains.
```

<div class="grid cards" markdown>

> -   :octicons-repo-16:{ .lg .middle } **Create a Multichain Tokens**

Reviewer 2: ```suggestion
-   :octicons-repo-16:{ .lg .middle } **Create Multichain Tokens**
```

    ---

>     Learn how to register your token on a source chain, register it on a target chain, and create a wrapped version for seamless interoperability.

Reviewer: ```suggestion
    Learn how to register your token on both a source and target chain, and create a wrapped version for seamless interoperability.
```

>     [:octicons-arrow-right-16: Start creating](/docs/tutorials/by-product/multichain-assets/multichain-token/)

Reviewer: ```suggestion
    [:octicons-arrow-right-16: Register your assets now](/docs/tutorials/by-product/multichain-assets/multichain-token/)
```
